src/disco/gui/fd_gui.c

```diff
@@ -453,11 +452,7 @@ fd_gui_txn_waterfall_snap( fd_gui_t *               gui,
     cur->out.tpu_quic_invalid += quic_metrics[ MIDX( COUNTER, QUIC_TILE, QUIC_PACKET_TOO_SMALL     ) ];
     cur->out.tpu_quic_invalid += quic_metrics[ MIDX( COUNTER, QUIC,      PKT_CRYPTO_FAILED         ) ];
     cur->out.tpu_quic_invalid += quic_metrics[ MIDX( COUNTER, QUIC,      PKT_NO_CONN               ) ];
-
-    cur->out.quic_frag_drop_g += quic_metrics[ MIDX( COUNTER, QUIC_TILE, TXNS_OVERRUN             ) ];
-    cur->out.quic_frag_drop   += quic_metrics[ MIDX( COUNTER, QUIC,      STREAM_CLOSED_DROP       ) ];
-    cur->out.quic_aborted     += quic_metrics[ MIDX( COUNTER, QUIC,      STREAM_CLOSED_CONN_ABORT ) ];
-    cur->out.quic_aborted     += quic_metrics[ MIDX( COUNTER, QUIC,      STREAM_CLOSED_PEER_RESET ) ];
```

Comment:
I kind of like having CONN_ABORT / PEER_RESET in the sankey for now ... cause it might reveal issues with the conn handling. We can remove a bit later once we are more confident?

Reply:
@mmcgee-jump Good point. We no longer generate these metrics in the same way. If a connection closes we no longer instantly reap all the lost streams. However we can generate these metrics lazily (when an eventual overrun happens)

Reply:
Probably but it would kind of be a layering violation (need to either shadow the conn table or add a "user data" region into the conn object)

Reply:
Done